omc3/correction/response_madx.py

```diff
@@ -321,7 +352,8 @@ def _add_coupling(dict_of_tfs: dict[str, tfs.TfsDataFrame]) -> dict[str, tfs.Tfs
     """
     result_dict_of_tfs = copy.deepcopy(dict_of_tfs)
     with timeit(lambda elapsed: LOG.debug(f"  Time adding coupling: {elapsed} s")):
-        for var, tfs_dframe in result_dict_of_tfs.items():  # already copies, so it's safe to act on them
+        # already copies, so it's safe to act on them
+        for var, tfs_dframe in result_dict_of_tfs.items():
```

Comment:
confusing now where this comment belongs to. put into loop and say dframe is a copy (or remove, the deepcopy is quite obv.)

also var -> _